src/main/java/com/SAS/League/Season.java

```diff
@@ -216,4 +210,24 @@ public void addRankPolicy(League league, LeagueRankPolicy rankPolicy) {
             league.addRankPolicy(this, rankPolicy);
         }
     }
+
+    /**
+     *
+     * @return the hashmap of the referees for this season for each league
+     */
+    public HashMap<League,HashSet<Referee>> getReferees() {
+        return referees;
+    }
+
+
+    /**
+     * @param league     the league that the  referee is relevant to
+     * @param ref the referee you want to add to the list of referees
+     */
+    public void addReferee(League league, Referee ref) {
+        if (this.leaguesList.contains(league)&&!this.rankPolicy.containsKey(league)) {
```

Comment:
why do you need to check in rank policy?

Reply:
i need to verify it doesn't have a rank policy yet.

Reply:
Why is it relates to the adding of the new referee?